grafana-plugin/src/pages/integrations/Integrations.tsx

```diff
@@ -347,7 +347,7 @@ class Integrations extends React.Component<IntegrationsProps, IntegrationsState>
       >
         <TextEllipsisTooltip placement="top" content={item.verbal_name}>
           <Text type="link" size="medium">
-            <Emoji className={cx('title', TEXT_ELLIPSIS_CLASS)} text={item.verbal_name} />
+            <Emoji className={cx(TEXT_ELLIPSIS_CLASS)} text={item.verbal_name} />
```

Comment:
AFAIR we use `u-`prefixed classes for such cases like `u-flex-center`

Reply:
TEXT_ELLIPSIS_CLASS was set to `overflow-child` before this PR. Not sure what you meant by this, there's a constant for it just for easier referencing.